### PR TITLE
[Validator] Mention `exactly` option for Length constraint

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fixes https://github.com/symfony/symfony-docs/issues/17059

### CODE REVIEW

**Review by alamirault — APPROVED** [review]

File: reference/constraints/Length.rst [hunk 1]
```diff
@@ -163,6 +163,21 @@ Parameter          Description
...
+This option is the exact length value. Validation will fail if
+the given value's length is not **exactly** equal to this value.
+
+.. note::
+
+    This option is the one being set by default when using the Length constraint
+    without passing any named argument to it. This means that for example,
+    ``@Assert\Length(20)`` and ``@Assert\Length(exactly=20)`` are equivalent, as
+    well as ``#[Assert\Length(20)]`` and ``#[Assert\Length(exactly: 20)]``.
+
```
Comment (OskarStark) [inline reference/constraints/Length.rst:180]: `versionadded` is missing

File: reference/constraints/Length.rst [hunk 2]
```diff
@@ -163,6 +163,21 @@ Parameter          Description
...
+**type**: ``integer``
+
+This option is the exact length value. Validation will fail if
+the given value's length is not **exactly** equal to this value.
+
+.. note::
+
+    This option is the one being set by default when using the Length constraint
+    without passing any named argument to it. This means that for example,
+    ``@Assert\Length(20)`` and ``@Assert\Length(exactly=20)`` are equivalent, as
```
Comment (OskarStark) [inline reference/constraints/Length.rst:178]: We can remove the annotation part in a follow up PR against `6.0`

**Review by OskarStark — APPROVED** [review]
After my comment

File: reference/constraints/Length.rst (hunk 1 above)
Comment (alexandre-daubois) [reply reference/constraints/Length.rst:180]: Updated!

File: reference/constraints/Length.rst (hunk 2 above)
Comment (alexandre-daubois) [reply reference/constraints/Length.rst:178]: I'll take care of this after the merge :ok_hand:

File: reference/constraints/Length.rst [hunk 3]
```diff
@@ -163,6 +163,25 @@ Parameter          Description
...
+    well as ``#[Assert\Length(20)]`` and ``#[Assert\Length(exactly: 20)]``.
+
+.. versionadded::
```
Comment (OskarStark) [inline reference/constraints/Length.rst]:
```suggestion
.. versionadded:: 5.2
```

**Comment by OskarStark** [comment]
Thank you Alexandre.

File: reference/constraints/Length.rst (hunk 3 above)
Comment (alexandre-daubois) [reply reference/constraints/Length.rst]: My bad :facepalm: Thanks again!

**Comment by OskarStark** [comment]
I already merged and added the version in https://github.com/symfony/symfony-docs/commit/fd74bd679368e41d287e4fdd0afc904984569903

**Pull request closed without merging**